jekyll/_cci2/env-vars.md

```diff
@@ -85,13 +85,14 @@ jobs:
 CircleCI does not support interpolation of environment variables.
 All defined values are treated literally.
 
-One workaround is to export the required variable within a command.
+If you have bash installed,
+you can export the required variable within a command.
 
 ```yaml
 - run:
+    name: Export Env Var
     command: |
-      export PATH=/go/bin:$PATH
-      go-get ...
+      echo 'export BAR=foo' >> $BASH_ENV
```

Comment:
Did the old line not work? Not sure what the point of using the "|" is for a single-line example -- showing adding something to the $PATH seemed more real-world.

Reply:
I only updated this because it was the most recent accepted example (based on a thread with Eric in #dx-team). I can see the real-world potential of the earlier example, and the multiline pipe is an oversight on my part. 😬